README.md

```diff
@@ -102,7 +102,7 @@ Explicit steps for setting up CCM in a development environment.
 10. Go To the `Settings` option from the Admin page --> Apps --> VIew App Configuration --> +App --> Choose Configuration type `By Client_id` --> Paste the client_id from Step 8
 11. Copy the Deployment_id by searching for the LTI tool you just added to the Canvas, Click setting button next it and right `click copy 'Deployment Id'`
 12. Run this command `docker exec -it ccm_web python manage.py rotate_keys`. This will create the Private and Public key for LTI
-13. Managing LTI registration to Tool: The LTI registration is completely handled from the Django Admin Console, so we need `UUID` from LTI Tool and `Client_id` and `Deployment_id` from the Canvas. So the Commandline option will help you in various use cases to set up LTI registration.  Please note *Issuer* and *Auth domain names* are different. For example, configuration in Canvas Test takes these values: `issuer=canvas.test.instructure.com` then `Auth URL=sso.test.canvaslms.com`. More [info](https://canvas.instructure.com/doc/api/file.lti_launch_overview.html).
+13. Managing LTI registration to Tool: The LTI registration is completely handled from the Django Admin Console, so we need `UUID` from LTI Tool and `Client_id` and `Deployment_id` from the Canvas. So the Commandline option will help you in various use cases to set up LTI registration.  Please note *Issuer* and *Auth domain names* are different. For example, configuration in Canvas Test takes these values: `issuer=canvas.test.instructure.com` then `Auth URL=sso.test.canvaslms.com`. If you're using a dev instance, then set `issuer=canvas.dev.instructure.com` and `Auth URL=sso.dev.canvaslms.com`. More [info](https://canvas.instructure.com/doc/api/file.lti_launch_overview.html).
```

Comment:
@PraveshKunwar, what we ended up using as below, and the canvas documentation link did not inform about it and hence we are adding a note here. Look at you DB table `lti_tool_ltiregistration` for your referance

issuer = canvas.instructure.com and Auth URL=sso.canvaslms.com